website/content/partials/telemetry-metrics/request-limiter-intro.mdx

```diff
@@ -1,2 +1,3 @@
-Request Limiter metrics relate to request success signals observed by the 
-request limiter and its current state.
\ No newline at end of file
+Request Limiter metrics relate to request success signals observed by the
+request limiter and its current state. Note the [request limiter is deprecated](/vault/docs/concepts/request-limiter)
```

Comment:
Why are we linking to the request limiter concept doc instead of the deprecation announcement page?

Reply:
This is sort of a unique case, which may not require a formal deprecation announcement. There are no user-facing changes and this feature was never taken out of beta (or enabled by default).

Let me/@banks know if you still think it should go in the deprecation announcement page, but I just went to add it and it felt out of place. Happy to be wrong though! 😄

Reply:
Hmm I linked this here because this page is the place that described that this beta feature has bee deprecated (or withdrawn I guess) in the warning box. We could also announce it in the upgrade guide/deprecations page. I'm open to it although as Mike pointed out this is a beta feature that was never actually GA so it's less clear cut if we should include it in a separate deprecation page.

Our Deprecation Notice page lists the 4 phases of deprecation. The first of these "supported" which means it was a GA feature working as intended. This feature never made it to that status so it's not the same as other things in that flow.

@schavis do you still think we should list it there?

Reply:
Fair enough. How do you feel about calling it out as an important change in the upgrade guide instead? As a general rule, we shouldn't be directly folks to content for deprecated features, which is why I'm hesitant to link to the request limiter page instead of one of the announcement pages.